Other/0CurrentDailyGifts.cs

```diff
@@ -46,87 +47,97 @@ public void GetAllGifts(DailyGiftsMode mode)
 
             #endregion
 
-            #region Expires 30-09-2022
-            GetGift(AvailableUntil(30, 09), "mythperception", "Boto", "Privateer's Cutlass", "Privateer's Dual Grenados", "Privateer's Grenado", "Privateer's Hair", "Privateer's Hand Cannon + Grenado", "Privateer's Locks");
-            //Pitiless Privateer Gear is in Trickster's Merge
-            GetGift(AvailableUntil(30, 09), "mythperception", "Corpo-Seco", "Cangaceiro's Carabina", "Cangaceiro's Crossed Peixeira", "Cangaceiro's Dual Peixeira", "Cangaceiro's Peixeira");
-            GetGift(AvailableUntil(30, 09), "mythperception", "Saci", "Cangaceira's Chapéu", "Cangaceira's Locks", "Cangaceiro Vitorioso", "Cangaceiro's Chapéu", "Cangaceiro's Chapéu + Beard", "Cangaceiro's Chapéu + Glasses");
-            GetGift(AvailableUntil(30, 09), "hiddendepths", "Aquamancer", "Crystal Staff of the Deep");
-            GetGift(AvailableUntil(30, 09), "blazebeard", "Pirate Captain", "Broken Split Blade of the Deep", "Split Blade of the Deep", "Split Blades of the Deep");
-            Bot.Quests.UpdateQuest(6033);
-            GetGift(AvailableUntil(30, 09), "celestialarenad", "Queen of Hope", "Arch Lich", "Arch Lich's Blessed Scrolls", "Arch Lich's Hood", "Arch Lich's Orb", "Arch Lich's Runes", "Arch Lich's Scrolls", "Arch Lich's Spear", "Arch Lich's Spears");
-            GetGift(AvailableUntil(30, 09), "deadmoor", "Nightmare", "Huasa's Cabello Elegante", "Huasa's Cabello Festejo", "Huasa's Cabello Oscuro", "Huasa's Rostro Elegante", "Huaso's Atuendo Elegante", "Huaso's Atuendo Escabroso", "Huaso's Atuendo Festejo", "Huaso's Atuendo Oscuro", "Huaso's Cabello Elegante", "Huaso's Cabello Festejo", "Huaso's Cabello Oscuro", "Huaso's Rostro Elegante");
-            GetGift(AvailableUntil(30, 09), "blazebeard", "Pirate Captain", "Pirate Captain Twig");
-            GetGift(AvailableUntil(30, 09), "hiddendepths", "Aqua Familiar", "Dagger of the Depths", "Daggers of the Depths", "Sword of the Depths", "Swords of the Depths");
+            // #region Expires 30-09-2022
 
-            #endregion
+            // GetGift(AvailableUntil(30, 09), "mythperception", "Boto", "Privateer's Cutlass", "Privateer's Dual Grenados", "Privateer's Grenado", "Privateer's Hair", "Privateer's Hand Cannon + Grenado", "Privateer's Locks");
```

Comment:
💯  fairly sure itll just skip it you don't need to comment them out

Reply:
I tried but it doesn't skip them, exactly on that part